### DISCUSSION TITLE
Listen specific action(s) instead of all

### DISCUSSION THREAD

**Discussion by RedskyThirty** [vuejs/pinia#1942] [Ideas]
It would be great if we could listen one or some specific actions instead of listening all with "$onAction".

Example:
```
export const useMyStore = defineStore('MyStore', {
	state: () => { return {}; },
	getters: {},
	actions: {
		sayHello() {},
		sayBonjour() {},
		sayHola() {},
	}
});
```
In a component:
`myStore.$onActions(['sayHello', 'sayHola'], (callback) => { });`

**Comment by posva** [comment]
I think that adding a new syntax to something that can be easily achieved with short code is not worth because it adds another thing to remember. Currently it can be achieved with:

```ts
myStore.$onActions(({ name }) => {
  if (name === 'sayHello' || name === 'sayHola') {}
});
```
